includes/classes/class-omise-transfers-table.php

```diff
@@ -56,13 +56,12 @@ function prepare_items() {
 
 		function get_columns() {
 			return $columns = array(
-				'trsf_amount'   => __( 'Amount' ),
-				'trsf_id'       => __( 'Transfer Id' ),
-				'trsf_sent'     => __( 'Sent' ),
-				'trsf_paid'     => __( 'Paid' ),
-				'trsf_failure'  => __( 'Failure Message' ),
-				'trsf_datetime' => __( 'Created' ),
-				'trsf_action'   => ''
```

Comment:
@nimid this one also (same as https://github.com/omise/omise-woocommerce/pull/24/files#r74709743)